pyblish_ftrack/plugins/integrate_ftrack_api.py

```diff
@@ -161,10 +161,18 @@ def process(self, instance):
                     collection = clique.parse(data["component_path"])
                 except ValueError:
                     # Assume its a single file
+                    # Changing file type
+                    component_entity["file_type"] = os.path.splitext(

```

Comment:
What happens when a file has no extension?

For safety, you could do something like this.

```python
name, ext = os.path.splitext(data["component_path"])
```

That way, it'll error out on a missing extension.